newIDE/app/src/ProjectsStorage/LocalProjectWriter.js

```diff
@@ -80,6 +134,38 @@ export default class LocalProjectWriter {
     }
   };
 
+  static saveProjectAs = (project: gdProject): Promise<void> => {
+    const filepath = project.getProjectFile();
+    const filenameWithExtension = path.basename(project.getProjectFile()) || "game.json";
+    const projectPath = path.dirname(project.getProjectFile());
+    const fileSystem = assignIn(new gd.AbstractFileSystemJS(), localFileSystem);
+    const browserWindow = electron.remote.getCurrentWindow();
+    const options = {
+    defaultPath: path.join(projectPath, filenameWithExtension),
+    filters: [{ name: 'GDevelop 5 project', extensions: ['json'] }],
+    };
+
+    if (!dialog) {
+    return Promise.reject('Unsupported');
+    }
+    const newFilepath = dialog.showSaveDialog(browserWindow, options);
+    if (!newFilepath || newFilepath === '') {
```

Comment:
nitpicking: !newFilePath will actually evaluate to false if the string is empty (in JavaScript, undefined, null, 0 and the empty string all evaluate to false when "casted" to a boolean), so you can just 
```suggestion
    if (!newFilepath) {
```